USER_GUIDE.md

```diff
@@ -359,7 +360,64 @@ try {
 
 ```
 
-## ClientBuilder
+## Client Factories
+
+You can create an OpenSearch Client using any [PSR-18](https://www.php-fig.org/psr/psr-18/) compatible 
```

Comment:
Since this is GitHub markdown maybe don't wrap it?
https://code.dblock.org/2021/06/07/to-wrap-or-not-to-wrap-in-markdown.html